docs/docs_source/fragments/no_lock_id.rst

```diff
@@ -0,0 +1,4 @@
+
+In the case where the file is locked by someone other than Office Online, hosts may not have a lock ID to return
+in the **X-WOPI-Lock** response header. The **X-WOPI-Lock** response header should be set to the empty string in
```

Comment:
Really? I thought the difference between Lock Mismatch and Locked By Another Interface was that for Lock Mismatch the X-WOPI-Lock header was present?